templates/workspaces/base/terraform/variables.tf

```diff
@@ -110,4 +110,8 @@ locals {
   vnet_subnets                   = cidrsubnets(var.address_space, 1, 1)
   services_subnet_address_prefix = local.vnet_subnets[0]
   webapps_subnet_address_prefix  = local.vnet_subnets[1]
+  tre_shared_service_tags = {
+    tre_id           = var.tre_id
+    tre_workspace_id = var.tre_resource_id
+  }
```

Comment:
You have this twice - here and in the new locals file you've added.
Since we have a bunch of locals here, I recommend you move the locals block from this file into in the one you've created.